pkg/aws/client.go

```diff
@@ -235,6 +236,46 @@ func (c *awsClient) ValidateCredentials() (bool, error) {
 	return true, nil
 }
 
+// Checks osdCcsAdmin IAM credentials are valid.
+// Get stack's key and actualy key and compare them
+// stack: aws cloudformation describe-stack-resource --logical-resource-id osdCcsAdminAccessKeys --stack-name osdCcsAdminIAMUser
+func (c *awsClient) ValidateCFUserCredentials() error {
+	name := AdminUserName
+	accessKeyInput := &iam.ListAccessKeysInput{
+		UserName: &name,
+	}
+	accessKeyList, err := c.iamClient.ListAccessKeys(accessKeyInput)
+	if err != nil {
+		return err
+	}
+
+	meta := accessKeyList.AccessKeyMetadata
+	keyList := make([]string, len(meta))
+	for _, key := range meta {
+		keyList = append(keyList, *key.AccessKeyId)
+	}
+
+	OsdCcsAdminStackNamePtr := OsdCcsAdminStackName
+	LogicalResourceIDPtr := "osdCcsAdminAccessKeys"
+	stackResourceInput := &cloudformation.DescribeStackResourceInput{
+		StackName:         &OsdCcsAdminStackNamePtr,
+		LogicalResourceId: &LogicalResourceIDPtr,
+	}
+	resources, err := c.cfClient.DescribeStackResource(stackResourceInput)
```

Comment:
is there a better way to get stack's user access key?

Reply:
@boranx unfortunately no. When you create an AWS IAM access key the secret access key is not saved and you cannot query the API for it